modules/persistent-storage-local-discovery.adoc

```diff
@@ -0,0 +1,167 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent-storage-local.adoc
+
+[id="local-storage-discovery_{context}"]
+= Automating discovery and provisioning for local storage devices
+
+The Local Storage Operator automates local storage discovery and provisioning. With this feature, you can simplify installation when dynamic provisioning is not otherwise available during deployment, such as with bare metal, VMware, or AWS store instances with attached devices.
+
+[IMPORTANT]
+====
+Automatic discovery and provisioning is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/.
+====
+
+Use the following procedure to create discovery requests, automatically discover local devices, and automatically provision local volumes for selected devices.
+
+.Prerequisites
+
+* The Local Storage Operator is installed.
+
+* Local disks are attached to {product-title} nodes.
+
+* You have access to the {product-title} web console and the `oc` command-line interface (CLI).
+
+.Procedure
+. To create a local volume discovery request from the web console:
+
+.. Navigate to *Operators* -> *OperatorHub* -> *Installed Operators*.
+
+.. Click on *Local Storage* from the Operators list.
+
+.. Select the *Local Volume Discovery Result* tab to display the list of results.
+
+.. Click *Create Local Volume Discovery Result*.
+
+.. Complete the required fields in *Form View* or click *YAML view* for full control.
```

Comment:
I am unsure as to whether 'click' is a recommended word. You could potentially replace 'click' with select or choose. There are other instances in the PR too.

Reply:
I'm not sure I'm always consistent about this, but I tend to use "click" when there's a UI button. 
ISG says:

> Use “click,” not “click on,” to refer to the action of selecting something on a graphical interface.